docs/build.adoc

```diff
@@ -208,13 +212,20 @@ build\windows\app\bin\Release\AMD64\vulkan_samples.exe
 * C{pp}14 Compiler
 * <<cmake-options,CMake Options>>
 * <<3d-models,3D models>>
+* Vulkan SDK
```

Comment:
Can this be reverted? We don't require the SDK to build our samples.

Reply:
Reverted